cmd/flags/migration.go

```diff
@@ -0,0 +1,35 @@
+package flags
+
+import (
+	"fmt"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	viper.SetDefault(MigrationSourceURLViperKey, migrationSourceURLDefault)
+	_ = viper.BindEnv(MigrationSourceURLViperKey, migrationSourceURLENV)
```

Comment:
do we need flags to specify where migrations are located? I cannot forseen users using differenet migration folder than the default one. If you cannot find specific use-cases for it I would suggest to remove this to prevent us to maintain it

Reply:
removed.